src/controllers/user.php

```diff
@@ -32,6 +30,9 @@ function changeExistingPassword($newPassword) {
         return $newPassword;
     }
 }
+function emptyFunction() {
+
+}
```

Comment:
` 
unwanted function
`

Reply:
we need this

Reply:
[eab7da4](https://github.com/kidistbezabih/TODO/pull/5/commits/eab7da4abcb84f2c843b8c9e5bbc31e3d22beedd)